addons/calendar/models/calendar_alarm_manager.py

```diff
@@ -226,6 +226,7 @@ def do_notif_reminder(self, alert):
 
         if alarm.alarm_type == 'notification':
             message = meeting.display_time
```

Comment:
Maybe use a f-string for this for easier readability:
```py
message = f'{meeting.display_time} <p>{"<br/>".join(alarm.body.split("\n")) if alarm.body else ""}</p>'
```

(Make sure to re-test it).

Reply:
Can't you simply escape the \ in f-string? like `"\\n"` ?
There must be a way to make it work.

And I don't understand your comment about "test_event_notification" :(

Reply:
done, since it's impossible to use a backslash in f-string expression, we use in place plaintext2html to format the body of the notification message.